src/js/components/agency/visualizations/StatusOfFundsChart.jsx

```diff
@@ -83,7 +85,7 @@ const StatusOfFundsChart = ({
         }, 50);
         window.addEventListener('resize', handleResize);
         return () => window.removeEventListener('resize', handleResize);
-    }, []);
```

Comment:
the window listener could be in a separate useeffect so we not adding and re-adding every time the window width changes.

Reply:
That change wasn't on purpose